src/test/java/juja/sqlcmd/ApplicationTest.java

```diff
@@ -0,0 +1,10 @@
+package juja.sqlcmd;
+
+import org.junit.Test;
+
+public class ApplicationTest {
+    @Test
+    public void testloadApplication(){
```

Comment:
Please pay attention to the open curly brace.
You should add one horizontal whitespace.

Reply:
Sure, I'll fix this point in next PR. Thanks for your remark!